airbyte-config/config-models/src/main/resources/types/SyncStats.yaml

```diff
@@ -29,3 +29,15 @@ properties:
     type: integer
   meanSecondsBetweenStateMessageEmittedandCommitted:
     type: integer
+  replicationStartTime: # the start of the replication activity
```

Comment:
could use the `description` key instead of comments :D

Reply:
Yup... sorted via bd0d8927d8d2e144515183192825ac11224f967a